### DISCUSSION TITLE
Bug: tells me to sponsor a project I'm already sponsoring

### DISCUSSION THREAD

**Discussion by 1player** [community/community#16767] [Other Feature Feedback, Questions, & Ideas]
When a project I sponsor appears on the feed, I get a "Sponsor" button, instead of the red heart "Sponsoring" button I get shown when navigating to the repo page. I thought I mistakingly I stopped donating to them, but it's just a *feeds* bug.

**Comment by mikekavouras** [comment]
@1player could I grab some additional info from you? Which developer are you sponsoring? Possibly a screenshot?

**Reply by 1player** [reply]
Found one example:

![image](https://user-images.githubusercontent.com/690120/172799958-4e9f42d6-04b7-4af4-92e2-645b615983d6.png)

And I'm a sponsor of them, as you can see in my profile.

**Reply by mikekavouras** [reply]
@1player a possible fix went out today. Assuming the event is still in your feed, would you mind giving it another look and reporting what you see? Thanks 🙏🏼

**Reply by 1player** [reply]
Looks like it's fixed now, thanks!
